yarn.lock

```diff
@@ -2652,85 +2652,85 @@
     estree-walker "^2.0.2"
     picomatch "^2.3.1"
 
-"@rollup/rollup-android-arm-eabi@4.21.3":
```

Comment:
are you using node 20..? I wonder the reason why of all these changes in yarn.lock